db/seeds.rb

```diff
@@ -477,6 +477,15 @@ def create_if_none(klass, count = nil)
   end
 end
 
+seeder.create_if_none(ListingEndorsement) do
+  5.times do
+    ListingEndorsement.create!(
+      content: Faker::Lorem.sentence,
+      user: User.find(rand(1...10)),
+      listing: Listing.find(rand(1...3)),
```

Comment:
This assumes that the IDs are always those but it's not necessarily through if one truncates the table with `rails db:seed:replant`

We should use the DB to fetch a random item, the same way we do in othe parts of the script:

```ruby
User.order(Arel.sql("RANDOM()")).first
```

for example

Reply:
oh got it